internal/syncd/integrations/ecr/listener.go

```diff
@@ -72,7 +71,8 @@ func (l *ImageListener) handler(r syncd.ImageReconciler) sqsconsumer.MessageHand
 		}
 
 		if !validImageTagRegex.MatchString(ecrEvent.Tag) {
-			return fmt.Errorf("malformed image tag")
+			l.logger.Log("error", "invalid image tag, the image will be ignored")
```

Comment:
This log message doesn't contain enough info to ever be actionable. Let's mention the name of the repo that contains the invalid image tag, as well as the invalid tag itself. A message like `ignoring event for invalid image tag "latest" found in repo "repo"` would be much more actionable.